caboose.go

```diff
@@ -53,10 +53,13 @@ const DefaultPoolLowWatermark = 5
 const DefaultSaturnRequestTimeout = 19 * time.Second
 const maxBlockSize = 4194305 // 4 Mib + 1 byte
 const DefaultOrchestratorEndpoint = "https://orchestrator.strn.pl/nodes/nearby?count=1000"
+const DefaultPoolRefreshInterval = 5 * time.Minute
 
 var ErrNotImplemented error = errors.New("not implemented")
 var ErrNoBackend error = errors.New("no available strn backend")
 var ErrBackendFailed error = errors.New("strn backend failed")
+var ErrContentNotFound error = errors.New("content not found")
```

Comment:
What Lassie is doing (returning 404) is semantically incorrect, will cause us headache when we will do GETs for graphs.
Let's make this clear that it is failure to find providers, and not failure to find path in requested path:

```suggestion
var ErrContentProviderNotFound error = errors.New("strn failed to find content providers")
```

I know this looks like a cosmetic difference, but trust me, one day, this will save someone hours in debugging :)

Reply:
Done.